lib/controller.dart

```diff
@@ -279,6 +279,17 @@ void _computeChartBuckets(List<model.Chart> charts) {
     }
   }
 
+  if (_dataNormalisationEnabled) {
+    for (var chart in charts) {
+      for (var chartCol in chart.fields) {
+        chartCol.bucket = [
+          (chartCol.bucket[0] * 100 / _filterValuesCount).truncate(),
```

Comment:
isn't this a bit problematic if we have small percentages for some filters? e.g. 0.8% or 2.9%?

What happens if we switch the buckets to have `num`s instead of `int`s so they can work with both counts and percentages?

Reply:
I actually couldn't get the num in the model generation to work - so took the shortcut to just use truncate. 

`Unhandled exception: failed to find type: num`

Reply:
right, looking at the model generation tool, it only supports `int`s... I think it would probably be good to add `num`s as a type at some point, so I've opened an issue there to keep track of this. In the meantime let's go with truncate, and can we add a little todo comment either here or in the model that the buckets should be `num` instead?